wagtail_bucketav/tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import wagtail_factories
+
+from wagtail_bucketav import utils
+
+
+@pytest.mark.django_db
+def test_get_image_with_key():
+    image = wagtail_factories.ImageFactory()
+    file_key = str(image.file)
+    discovered_object = utils.get_object_for_key(file_key)
```

Comment:
Nitpick:
```suggestion
    result = utils.get_object_for_key(file_key)
```

Reply:
Is this better?

Reply:
Tests normally follow a few standard patterns. Where it is the result of a function/method/process is being tested, I try to always use `result` for the result, and will usually check that against an `expected_result` for correctness (not applicable in this situation, but just saying). It's marked as a nitpick, so not hugely important :)